packages/purpose-template-process/src/services/validators.ts

```diff
@@ -0,0 +1,77 @@
+import { RiskAnalysisFormTemplate, TenantKind } from "pagopa-interop-models";
+import { purposeTemplateApi } from "pagopa-interop-api-clients";
+import {
+  RiskAnalysisTemplateValidatedForm,
+  riskAnalysisValidatedFormTemplateToNewRiskAnalysisFormTemplate,
+  validatePurposeTemplateRiskAnalysis,
+} from "pagopa-interop-commons";
+import {
+  missingFreeOfChargeReason,
+  purposeTemplateNameConflict,
+  riskAnalysisTemplateValidationFailed,
+} from "../model/domain/errors.js";
+import { ReadModelServiceSQL } from "./readModelServiceSQL.js";
+
+export const assertConsistentFreeOfCharge = (
+  isFreeOfCharge: boolean,
+  freeOfChargeReason: string | undefined
+): void => {
+  if (isFreeOfCharge && !freeOfChargeReason) {
+    throw missingFreeOfChargeReason();
+  }
+};
+
+export const assertPurposeTemplateTitleIsNotDuplicated = async ({
+  readModelService,
+  title,
+}: {
+  readModelService: ReadModelServiceSQL;
+  title: string;
+}): Promise<void> => {
+  const purposeTemplateWithSameName = await readModelService.getPurposeTemplate(
+    title
+  );
+  const purposeTemplateId = purposeTemplateWithSameName?.data.id;
+  if (purposeTemplateWithSameName && purposeTemplateId) {
+    throw purposeTemplateNameConflict(purposeTemplateId, title);
+  }
+};
+
+export function validateAndTransformRiskAnalysisTemplate(
+  purposeRiskAnalysisForm:
+    | purposeTemplateApi.RiskAnalysisFormTemplateSeed
+    | undefined,
+  tenantKind: TenantKind
+): RiskAnalysisFormTemplate | undefined {
+  if (!purposeRiskAnalysisForm) {
+    return undefined;
+  }
+
+  const validatedForm = validateRiskAnalysisTemplateOrThrow({
+    riskAnalysisForm: purposeRiskAnalysisForm,
+    tenantKind,
+  });
+
+  return riskAnalysisValidatedFormTemplateToNewRiskAnalysisFormTemplate(
+    validatedForm
+  );
+}
+
+export function validateRiskAnalysisTemplateOrThrow({
```

Comment:
```suggestion
function validateRiskAnalysisTemplateOrThrow({
```
It seems no need to export